components/layout_2020/style_ext.rs

```diff
@@ -395,7 +395,11 @@ impl ComputedValuesExt for ComputedValues {
         !self.get_position().z_index.is_auto()
     }
 
-    fn establishes_containing_block(&self) -> bool {
+    /// Returns true if this style establishes a containing block for absolute
+    /// descendants. This will also return true if it establishes a containing
+    /// block for all descendants, which include those which are both
+    /// fixed and absolutely positioned.
```

Comment:
i think these can be made a bit clearer. how about

```rust
    /// Returns true if this style establishes a containing block for absolute
    /// descendants (position: absolute or fixed). This will also return true if
    /// it establishes a containing block for all descendants, because “all
    /// descendants” includes absolute descendants.
```

and in the all descendants method

```rust
    /// Returns true if this style establishes a containing block for all descendants,
    /// not just absolute descendants (position: absolute or fixed).
```

Reply:
Thanks for the review! I have adjusted these a bit because `establishes_containing_block_for_absolute_descendants()` doesn't imply that a containing block is establishes for `position: fixed` descendants. 

The new comments are:

```
    /// Returns true if this style establishes a containing block for absolute
    /// descendants (`position: absolute`). If this style happens to establish a
    /// containing block for “all descendants” (ie including `position: fixed`
    /// descendants) this method will return true, but a true return value does
    /// not imply that the style establishes a containing block for all descendants.
    /// Use `establishes_containing_block_for_all_descendants()` instead.
    fn establishes_containing_block_for_absolute_descendants(&self) -> bool {
```

```
    /// Returns true if this style establishes a containing block for
    /// all descendants, including fixed descendants (`position: fixed`).
    /// Note that this also implies that it establishes a containing block
    /// for absolute descendants (`position: absolute`).
     fn establishes_containing_block_for_all_descendants(&self) -> bool {
```